### ISSUE TITLE
Cyrillic revision

### ISSUE THREAD

**Issue by den32ds** [Fuzzypeg/Coelacanth#16]
Awesome typeface, but Cyrillic is bad for now

**Comment by Fuzzypeg** [comment]
Yep, it's pretty bad. This repo is moving to https://gitlab.com/Fuzzypeg/coelacanth so I'll recreate the ticket there.
